pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -292,18 +322,56 @@ process runIqtree {
     """
 }
 
+// Function to check if a genome is present in
+// the annotation cache.
+def annoInCache(genome, anno_cache) {
+    genome = new File("$genome")
+    genome = genome.getName()
+    File file = new File("${anno_cache}/${genome}/annotation.gtf")
+    return file.exists()
+}
+
 workflow {
     println(ensemblLogo())
+
+    // Prepare busco protein set:
     prepareBusco(params.busco_proteins)
+
+    // Prepare input genomes:
     genomes = Channel.fromPath("${params.dir}/*.*")
     prepareGenome(genomes)
 
-    buscoAnnot(prepareBusco.out.busco_prots, prepareGenome.out.proc_genome)
-    runGffread(buscoAnnot.out.busco_annot, buscoAnnot.out.genome)
+    // Branch the genomes channel based on presence in the
+    // annotation cache:
+    prepareGenome.out.proc_genome.branch {
+        annot: !annoInCache(it, params.anno_cache)
+        link: annoInCache(it, params.anno_cache)
+    }.set { genome_fork }
+
+    // Link annotations present in the cache:
```

Comment:
If you decide to put this description in the Groovydoc, I'd suggest to remove it from here.
```suggestion
```